### PR TITLE
USHIFT-367: apply rebase.sh to service-ca manifests

### PR DESCRIPTION
**Pull request by fzdarsky**
Updates the service-ca manifests using the latest rebase.sh automation.

Most changes are completely neutral and result from the adoption of yq, e.g. reordering of fields resulting from yq appending added fields at the end rather than ordering them in alphabetically or different indentation. A few changes drop comments or labels that are no longer applied by the respective operator.

Signed-off-by: Frank A. Zdarsky [fzdarsky@redhat.com](mailto:fzdarsky@redhat.com)

Partially addresses [USHIFT-367](https://issues.redhat.com//browse/USHIFT-367)

### CODE REVIEW

File: assets/components/service-ca/deployment.yaml [hunk 1]
```diff
@@ -23,45 +22,45 @@ spec:
...
-        resources:
-          requests:
-            memory: 120Mi
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /var/run/secrets/signing-key
-          name: signing-key
-        - mountPath: /var/run/configmaps/signing-cabundle
-          name: signing-cabundle
+        - name: service-ca-controller
```
Comment (stlaz) [inline assets/components/service-ca/deployment.yaml:27]: this whitespace is not present in https://github.com/openshift/service-ca-operator/blob/master/bindata/v4.0.0/controller/deployment.yaml

File: assets/components/service-ca/deployment.yaml [hunk 2]
```diff
@@ -23,45 +22,45 @@ spec:
...
-      # nodeSelector:
-      #   node-role.kubernetes.io/master: ""
+        - name: signing-key
+          secret:
+            secretName: {{.TLSSecret}}
+        - name: signing-cabundle
+          configMap:
+            name: {{.CAConfigMap}}
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
```
Comment (stlaz) [inline assets/components/service-ca/deployment.yaml:52]: It's unlikely that this was commented out in our code, you may want to figure out why this was here.

File: assets/components/service-ca/deployment.yaml [hunk 3]
```diff
@@ -23,45 +22,45 @@ spec:
...
-        effect: "NoSchedule"
-      - key: "node.kubernetes.io/unreachable"
-        operator: "Exists"
-        effect: "NoExecute"
-        tolerationSeconds: 120
-      - key: "node.kubernetes.io/not-ready"
-        operator: "Exists"
-        effect: "NoExecute"
-        tolerationSeconds: 120
+        - key: node-role.kubernetes.io/master
```
Comment (stlaz) [inline assets/components/service-ca/deployment.yaml:55]: again - these whitespaces don't appear in the orig

Comment (fzdarsky) [reply assets/components/service-ca/deployment.yaml:55]: The whitespace change is the result from `yq`'s reformatting.

File: assets/components/service-ca/deployment.yaml (hunk 1 above)
Comment (fzdarsky) [reply assets/components/service-ca/deployment.yaml:27]: The whitespace change is the result from `yq`'s reformatting.

File: assets/components/service-ca/deployment.yaml (hunk 2 above)
Comment (fzdarsky) [reply assets/components/service-ca/deployment.yaml:52]: We originally didn't have node roles set; this was fixed in #869 only recently; we probably missed removing the comment.

**Comment by dhellmann** [comment]
Is "the latest rebase automation" the version in the repo now, or in another PR?

**Pull request merged**